core/src/main/java/org/apache/iceberg/jdbc/JdbcCatalog.java

```diff
@@ -199,21 +198,29 @@ private void initializeCatalogTables() throws InterruptedException, SQLException
         });
   }
 
-  private void updateCatalogTables(Connection connection) throws SQLException {
-    DatabaseMetaData dbMeta = connection.getMetaData();
-    ResultSet typeColumn =
-        dbMeta.getColumns(null, null, JdbcUtil.CATALOG_TABLE_VIEW_NAME, JdbcUtil.RECORD_TYPE);
-    if (typeColumn.next()) {
-      LOG.debug("{} already supports views", JdbcUtil.CATALOG_TABLE_VIEW_NAME);
-    } else {
-      if (PropertyUtil.propertyAsBoolean(
-          catalogProperties, JdbcUtil.ADD_VIEW_SUPPORT_PROPERTY, false)) {
-        connection.prepareStatement(JdbcUtil.UPDATE_CATALOG_SQL).execute();
-      } else {
-        LOG.warn(VIEW_WARNING_LOG_MESSAGE);
-        schemaVersion = JdbcUtil.SchemaVersion.V0;
-      }
-    }
+  private void checkAndMaybeUpdateSchema() throws SQLException, InterruptedException {
+    LOG.trace(
+        "Checking the SQL schema version and eventually update if {} is true",
+        JdbcUtil.ADD_VIEW_SUPPORT_PROPERTY);
```

Comment:
There is no need for a log message here. All this is doing is logging that this method was called. That's information that comes from a stack trace if needed.

Reply:
OK, I just mimic the same as in `initializeCatalogTables()` method.
Let me remove it (maybe worth to remove the `trace` in `initializeCatalogTables()` as well.